datasources/src/main/scala/com/normation/plugins/datasources/Serialization.scala

```diff
@@ -417,7 +417,7 @@ trait DataSourceExtractor[M[_]] extends JsonExctractorUtils[M] {
 }
 
 object DataSourceExtractor {
-  object OptionnalJson extends DataSourceExtractor[Option] {
+  object OptionalJson extends DataSourceExtractor[Option] {
```

Comment:
Is it really ok to rename it? I'm not sure it canot be used somewhere else

Reply:
you need to change it where it's used, too (like you did) but appart that, it's ok

Reply:
since datasources is a plugin (and plugin can't depend of other plugin for now), it's safe

Reply:
:+1: